deploy/deployment_dev_cluster.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: registration-service-v0.1
+  namespace: toolchain-registration-service
+spec:
+  lookupPolicy:
+    local: false
+  tags:
+  - from:
+      kind: DockerImage
+      name: registry.svc.ci.openshift.org/codeready-toolchain/registration-service-v0.1:registration-service
+    importPolicy:
+      scheduled: true
+    name: registration-service
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  labels:
+    app: registration-service
+  name: registration-service
+  namespace: toolchain-registration-service
```

Comment:
If we deploy registration service to this namespace, then we'll be creating sa, role and rolebinding in the namespace. This service account used in registration service will create usersignup custom resource, but if host-operator is not watching the same namespace for required usesignup resource, then it will fail to reconcile